js/navigation/NavigationLayout.js

```diff
@@ -28,7 +50,7 @@ const ProfileStack = createStackNavigator(
       ...sharedNavigationOptions(navigation)
     })
   }
-);
+)
 const EditProfileStack = createStackNavigator(
```

Comment:
this stack is not being used. Why? If you don't need it then remove it, please.

Reply:
that's devin's stack, not sure why it's not being used...

Reply:
devin removed it